charts/ccsm-helm/charts/identity/templates/_helpers.tpl

```diff
@@ -0,0 +1,54 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "identity.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Defines extra labels for operate.
+*/}}
+{{- define "identity.extraLabels" -}}
+app.kubernetes.io/component: identity
+{{- end -}}
+
+{{/*
+Define common labels for operate, combining the match labels and transient labels, which might change on updating
```

Comment:
```suggestion
Define common labels for identity, combining the match labels and transient labels, which might change on updating
```

Reply:
@Zelldon it still says `operate` instead of `identity` in this file, may be you forgot to update these lines?